package/MDAnalysis/analysis/md.bib

```diff
@@ -0,0 +1,23 @@
+@article{[Best2013],
```

Comment:
I'd suggest not using unnecessary punctuation in the citation key, I feel like it could confuse the extension.